recipes/libgd/meta.yaml

```diff
@@ -14,7 +14,7 @@ requirements:
     - jpeg >=9* # use recent libjpeg
     - libpng
     - libtiff
-    - fontconfig # for libfontconfig.so
+    - fontconfig >=2.11 # for libfontconfig.so
     
   run:
     - libgcc # [not osx]
```

Comment:
Mh ... http://jinja.pocoo.org/docs/dev/templates/#line-statements
```
Since Jinja 2.2, line-based comments are available as well. For example, if the line-comment prefix is configured to be ##, everything from ## to the end of the line is ignored (excluding the newline sign):
```